sdbuild/scripts/create_bsp.sh

```diff
@@ -6,6 +6,14 @@ set -e
 board=$1
 template=$2
 
+# Allow fetching the BSP from a URL (useful for CI integration)
```

Comment:
If possible I'd split this out into a separate PR so that we get a clearer history in the repository

Reply:
Yeah, I hadn't intended to include this in the PR but github automagically included it as I pushed my working tree.  I'll split it out.